src/librustc/mir/interpret/error.rs

```diff
@@ -264,6 +264,8 @@ pub enum PanicInfo<O> {
     RemainderByZero,
     GeneratorResumedAfterReturn,
```

Comment:
We could also just have two variants `ResumedAfterReturn(GeneratorKind)` and `ResumedAfterPanic(GeneratorKind)`

Reply:
Ok I will experiment with that idea and see what it looks like.